tools/gen_api_type_database.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -e
+
+# Clean up any stale files in the API tree output. Bazel remembers valid cached
+# files still.
+rm -rf bazel-bin/external/envoy_api
+
+# TODO(htuch): This script started life by cloning docs/build.sh. It depends on
+# the @envoy_api//docs:protos target in a few places as a result. This is not
+# guaranteed to be the precise set of protos we want to format, but as a
+# starting place it seems reasonable. In the future, we should change the logic
+# here.
+bazel build ${BAZEL_BUILD_OPTIONS} --//tools/api_proto_plugin:default_type_db_target=@envoy_api//docs:protos \
```

Comment:
This doesn't generate type_database outside of bazel cache. I guess because you already have locally generated one so the build just works.

Reply:
This still doesn't, just roll this script back to proto_format?